static/src/js/components/ChunkedOrderModal/ChunkedOrderModal.js

```diff
@@ -13,97 +13,95 @@ import { locationPropType } from '../../util/propTypes/location'
-          {
-            pathname: '/search',
-            search: stringify({
-              ...params,
-              p
-            })
-          }
-        }
-        updatePath
-      >
-        Refine your search
-      </PortalLinkContainer>
-    )
-
-    const body = (
-      <>
+  // Remove focused collection from back button params
+  const params = parse(location.search, {
+    ignoreQueryPrefix: true,
+    parseArrays: false
+  })
+  let { p = '' } = params
+  p = p.replace(/^[^!]*/, '')
+
+  const backLink = (
+    <PortalLinkContainer
+      className="chunked-order-modal__action chunked-order-modal__action--secondary"
+      bootstrapVariant="primary"
+      type="button"
+      icon={FaArrowCircleLeft}
+      label="Refine your search"
+      onClick={onModalClose}
+      to={
         {
-          Object.keys(projectCollectionsRequiringChunking).length > 0 && (
-            Object.keys(projectCollectionsRequiringChunking).map((collectionId, i) => {
-              const key = `chunked_order_message-${i}`
-
-              const { [collectionId]: projectCollection } = projectCollectionsRequiringChunking
-              const orderCount = calculateOrderCount(projectCollection)
-
-              const {
-                accessMethods = {},
-                granules = {},
-                selectedAccessMethod
-              } = projectCollection
-
-              const { hits: granuleCount } = granules
-
-              const granulesPerOrder = calculateGranulesPerOrder(
-                accessMethods,
-                selectedAccessMethod
-              )
-
-              const { [collectionId]: projectCollectionMetadata = {} } = projectCollectionsMetadata
-              const { title } = projectCollectionMetadata
-
-              return (
-                <p
-                  key={key}
-                  data-testid={key}
-                >
+          pathname: '/search',
+          search: stringify({
+            ...params,
+            p
+          })
+        }
+      }
+      updatePath
+    >
+      Refine your search
+    </PortalLinkContainer>
+  )
+  const body = (
+    <div>
+      {
+        Object.keys(projectCollectionsRequiringChunking).length > 0 && (
+          Object.keys(projectCollectionsRequiringChunking).map((collectionId, i) => {
+            const key = `chunked_order_message-${i}`
+
+            const { [collectionId]: projectCollection } = projectCollectionsRequiringChunking
+            const orderCount = calculateOrderCount(projectCollection)
+
+            const {
+              accessMethods = {},
+              granules = {},
+              selectedAccessMethod
+            } = projectCollection
+
+            const { hits: granuleCount } = granules
+
+            const granulesPerOrder = calculateGranulesPerOrder(
+              accessMethods,
+              selectedAccessMethod
+            )
+
+            // Display notice for services which send confirmation emails
+            let sendsEmails = false
```

Comment:
@eudoroolivares2016 We wanna do this stuff ONLY for echo and ESI? We don't want to just exclude harmony orders?